### PR TITLE
Bugfix and Change

### PR DESCRIPTION
**Pull request by Bletch1971**
1. Add a fix for the Message of the Day to output multiple lines correctly.
2. SystemIniFile.cs - added Multiline Attribute and changed Serialize and
   Deserialize to handle NewLine characters
3. ServerProfile.cs - added Multiline = true to the MOTD property
4. Added two Max XP reset buttons, one for player section, one for dino section.
5. en-US.xaml - added generic tooltip for the button
6. ServerProfile.cs - added 2 constants values for MaxXP, updated
   DependancyProperties to use constant values and added 2 new methods for
   resetting the values.
7. ServerSettingsControl.xaml - added two new reset buttons to the player
   and dino section next to the Max XP slider. Also added margins to the
   other sliders to align them.

### CODE REVIEW

**Comment by Bletch1971** [comment]
This is what the reset buttons look like

playerreset

dinoreset

File: ARK Server Manager/Globalization/en-US/en-US.xaml [hunk 1]
```diff
@@ -290,6 +290,7 @@
     <sys:String x:Key="ServerSettings_MaxXPLabel">Max XP:</sys:String>
     <sys:String x:Key="ServerSettings_MaxXPTooltip">Set the maximum XP cap.</sys:String>
     <sys:String x:Key="ServerSettings_MaxXPUnits">xp</sys:String>
+    <sys:String x:Key="ServerSettings_ResetMaxXP">Reset Max XP Cap to official default</sys:String>
```
Comment (ChronosWS) [inline ARK Server Manager/Globalization/en-US/en-US.xaml]: *Tooltip

File: ARK Server Manager/Lib/Model/ServerProfile.cs [hunk 2]
```diff
@@ -25,7 +25,10 @@ public interface ISettingsBag
     [XmlRoot("ArkServerProfile")]
     [Serializable()]
     public class ServerProfile : DependencyObject
-    { 
+    {
+        private const int DEFAULT_MAX_EXPERIENCE_POINTS_DINO = 275000;
```
Comment (ChronosWS) [inline ARK Server Manager/Lib/Model/ServerProfile.cs]: These should go in GameData and can be public.

**Comment by Bletch1971** [comment]
Corrections made, happy to change anything else if needed.

**Comment by Bletch1971** [comment]
I would like to add a reset button to most of the sections, to reset that sections values, but it would mean a lot of new constant values being added to GameData. The reset button would be at the top of the section and resets all values in that section.

Are you OK with this?

**Comment by ChronosWS** [comment]
I don't have a problem with it.  Is there a defaults file that comes with
the server now though?

**Comment by Bletch1971** [comment]
No defaults file that I know of, I was just going to reset the section values back to the defaults you use when you create a new profile.
I have tried in the past to get default values for all the server settings used by ARK, but nothing is ever complete or explained properly on Steam or the wiki.
Getting server setting information is painful.

**Pull request merged**